tracer/ext/priority.go

```diff
@@ -0,0 +1,19 @@
+package ext
+
+// Priority is a hint given to the backend so that it knows which traces to reject or kept.
+// In a distributed context, it should be set before any context propagation (fork, RPC calls) to be effective.
+
+const (
+	// PriorityUserReject informs the backend that a trace should be rejected and not stored.
+	// This should be used by user code overriding default priority.
+	PriorityUserReject = -1
+	// PriorityAutoReject informs the backend that a trace should be rejected and not stored.
+	// This is used by the builtin sampler.
+	PriorityAutoReject = 0
+	// PriorityAutoReject informs the backend that a trace should be kept and not stored.
```

Comment:
@ufoot Looks like the name here (`PriorityAutoReject`) doesn't match the description (`PriorityAutoKeep`)? Typo?